presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoRpmController.java

```diff
@@ -17,6 +17,7 @@
 import com.google.gson.JsonObject;
 import com.google.gson.JsonParser;
 import com.google.inject.Inject;
+import com.teradata.prestomanager.common.StopType;
```

Comment:
Since `Controller` already have dependency on `common` module, we move that from agent to common so that we could use it in `Controller` module